tests/test_application.py

```diff
@@ -24,242 +24,253 @@ def GET(self):
 
 """
 
-urls = (
-    "/iter", "do_iter",
-)
+urls = ("/iter", "do_iter")
 app = web.application(urls, globals())
 
+
 class do_iter:
     def GET(self):
-        yield 'hello, '
-        yield web.input(name='world').name
+        yield "hello, "
+        yield web.input(name="world").name
 
     POST = GET
 
+
 def write(filename, data):
-    f = open(filename, 'w')
+    f = open(filename, "w")
     f.write(data)
     f.close()
 
+
 class ApplicationTest(unittest.TestCase):
     def test_reloader(self):
-        write('foo.py', data % dict(classname='a', output='a'))
+        write("foo.py", data % dict(classname="a", output="a"))
         import foo
+
         app = foo.app
 
-        self.assertEqual(app.request('/').data, b'a')
+        self.assertEqual(app.request("/").data, b"a")
 
         # test class change
         time.sleep(1)
-        write('foo.py', data % dict(classname='a', output='b'))
-        self.assertEqual(app.request('/').data, b'b')
+        write("foo.py", data % dict(classname="a", output="b"))
+        self.assertEqual(app.request("/").data, b"b")
 
         # test urls change
         time.sleep(1)
-        write('foo.py', data % dict(classname='c', output='c'))
-        self.assertEqual(app.request('/').data, b'c')
+        write("foo.py", data % dict(classname="c", output="c"))
+        self.assertEqual(app.request("/").data, b"c")
 
     def testUppercaseMethods(self):
         urls = ("/", "hello")
         app = web.application(urls, locals())
+
         class hello:
-            def GET(self): return "hello"
-            def internal(self): return "secret"
+            def GET(self):
+                return "hello"
```

Comment:
Looks like the `internal` function disappeared. Bug in Black?

Reply:
It just moved down...  More like a bug in __diff__.

Reply:
The diff is correct.